code/modules/world_api/commands/server_management.dm

```diff
@@ -120,3 +120,19 @@
 	statuscode = 200
 	response = "Text sent"
 	return TRUE
+
+//Reloads all admins via remote command. Updates from the forumuser API if enabled.
+/datum/topic_command/admins_reload
+	name = "admins_reload"
+	description = "Reloads all and pulls new data from the forumuser API if it's enabled."
+	params = list(
+		"senderkey" = list("name"="senderkey","desc"="A display friendly name for the sender.","req"=1,"type"="str")
+	)
+
+/datum/topic_command/broadcast_text/run_command(queryparams)
+	log_and_message_admins("[queryparams["senderkey"]] initiated a remote reload of the admins list.")
+
+	if (config.use_forumuser_api)
+		update_admins_from_api(reload_once_done=FALSE)
+
+	load_admins()
```

Comment:
Would imho be good to return a different status code / message in these two cases.

Reply:
I can't think of a good status code between the two? Both would be 200. Status message is doable I guess.

Reply:
I´d suggest 201 if the forumuserapi has been used and 200 otherwise.
But anything in the 200 range would imho be fine.
(We dont claim to stick to the http standard with the status codes anyway)

Reply:
Addressed. Also added a failure case for when pestering the API for a reload fails.